src/slices/HeadlineWithButton.js

```diff
@@ -1,18 +1,13 @@
 import * as React from 'react'
 import { graphql } from 'gatsby'
-import { RichText } from 'prismic-reactjs'
+import { PrismicRichText } from '@prismicio/react'
 import { GatsbyImage } from 'gatsby-plugin-image'
 
-import { GatsbyLink } from '../components/GatsbyLink'
-
 export const HeadlineWithButton = ({ slice }) => (
   <section className="headline-with-button">
     <div>
-      <RichText render={slice.primary.headline.richText || []} />
-      <RichText
-        render={slice.primary.description.richText || []}
-        serializeHyperlink={GatsbyLink}
```

Comment:
I assume the `internalLinkComponent` in `PrismicProvider` takes care of these links from now on, is that correct?

Reply:
Correct; `<PrismicRichText>` will automatically use `internalLinkComponent` and `linkResolver` given to `<PrismicProvider>`. Users can still override these manually by providing them to `<PrismicRichText>`, but that would be an uncommon use case.

Reply:
`<PrismicRichText>` also automatically handles null Rich Text fields (which is why `|| []` is no longer needed). `<PrismicRichText>` will render `null` in that case.